src/patternfly/components/Button/docs/code.md

```diff
@@ -4,21 +4,22 @@ Always add a modifier class to add color to the button.
 
 ## Accessibility
 
-An active state button will appear pressed when active. There’s no need to add a class to `<button>`s as they use a pseudo-class. However, if you are using a button as a toggle, you can still force the same active appearance with `pf-is-active` (and include the `aria-pressed="true"`) should you need to replicate the state programmatically.
-
-Disabled buttons should include the `disabled="disabled"` attribute to indicate the state of the element to assistive technologies.
+| Attribute | Applied | Outcome |
+| -- | -- | -- |
+| `disabled="disabled"` | `.pf-c-button` | Indicates the disabled state of the element to assistive technologies |
```

Comment:
Something to note about this attribute, is that we we expand this component so that these styles can be applied to `<a>`, then we would want to specify that this attribute applies to the `<button>` element only. In the menu, I include both buttons and links as menu item examples, and handled the different attributes as shown here: https://deploy-preview-206--pf-next.netlify.com/components/Dropdown/examples/#accessibility

I realize this comment is not part of the scope of this PR, but I wonder if we should at least mention the `<button>` element somewhere in this row?